### PR TITLE
Adventure Console Content Update 1

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
Adds a few limbus company events to the adventure console and one new general event.
Additionally adds defines for balancing coin and stat rewards.

So far general events is just me shooting in the wind about feelings i have or just writing practice.

## Why It's Good For The Game
More content for adventure console.

## Changelog
:cl:
add: new adventure console events
/:cl:

### CODE REVIEW

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/monkey_tree.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		if(4)
+			CHANCE_BUTTON_FORMAT(10, "SPRINKLE MORE WATER", M)
+			BUTTON_FORMAT(5, "SNAP THE TWIG", M)
+			. += CoinFlipping(M)
+			return
+		if(5)
+			AdjustHitPoint(-10)
+			AdjustStatNum(PRIDE_STAT, ADV_EVENT_STAT_EASY)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/monkey_tree.dm:62]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/lost_mind.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		if(3)
+			CHANCE_BUTTON_FORMAT(ReturnStat(GLOOM_STAT), "GLOOM", M)
+			CHANCE_BUTTON_FORMAT(ReturnStat(GLUTT_STAT), "GLUTTONY", M)
+			CHANCE_BUTTON_FORMAT(ReturnStat(PRIDE_STAT), "PRIDE", M)
+			. += CoinFlipping(M)
+			return
+		if(5)
+			AdjustHitPoint(-13)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/lost_mind.dm:55]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/kqe.dm [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+			BUTTON_FORMAT(4, "CONFESS THE TRUTH", M)
+			BUTTON_FORMAT(5, "TELL IT THAT YOU TOOK A TOUR", M)
+			BUTTON_FORMAT(7, "LEAVE WITHOUT ANSWERING", M)
+			return
+		if(5)
+			AdjustCurrency(ADV_EVENT_COIN_EASY)
+		if(6)
+			AdjustHitPoint(-15)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/kqe.dm:76]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/golden_grip.dm [hunk 4]
```diff
@@ -53,8 +53,8 @@
 			.+= CoinFlipping(M)
 			return
 		if(6)
-			AdjustCurrency(3)
+			AdjustCurrency(ADV_EVENT_COIN_EASY)
 		if(7)
-			AdjustCurrency(1)
+			AdjustCurrency(ADV_EVENT_COIN_EASY)
 			AdjustHitPoint(-15)
 	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/golden_grip.dm:60]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/fourthfront_smokewar.dm [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+		if(5)
+			AdjustHitPoint(-1)
+			CHANCE_BUTTON_FORMAT(ReturnStat(LUST_STAT), "LUST", M)
+			CHANCE_BUTTON_FORMAT(ReturnStat(PRIDE_STAT), "PRIDE", M)
+			.+= CoinFlipping(M)
+			return
+		if(6)
+			AdjustHitPoint(-10)
+			AdjustCurrency(ADV_EVENT_COIN_EASY)
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/fourthfront_smokewar.dm:70]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/chicken_strife.dm [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+			return
+		if(2)
+			AdjustStatNum(WRATH_STAT, ADV_EVENT_STAT_EASY)
+			AdjustStatNum(GLUTT_STAT, ADV_EVENT_STAT_EASY)
+		if(3)
+			AdjustStatNum(WRATH_STAT, ADV_EVENT_STAT_EASY)
+		if(4)
+			AdjustStatNum(GLUTT_STAT, ADV_EVENT_STAT_EASY)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/chicken_strife.dm:53]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/blubbering_toad.dm [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+		if(2)
+			CHANCE_BUTTON_FORMAT(ReturnStat(GLOOM_STAT), "GLOOM", M)
+			. += CoinFlipping(M)
+			return
+		if(4)
+			AdjustHitPoint(5)
+		if(5)
+			AdjustStatNum(PRIDE_STAT, ADV_EVENT_STAT_NORMAL)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/blubbering_toad.dm:59]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/ardor_blossom.dm [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+	switch(cords)
+		if(1)
+			BUTTON_FORMAT(2, "REACH OUT", M)
+			BUTTON_FORMAT(5, "TURN AROUND", M)
+			return
+		if(2)
+			AdjustHitPoint(-10)
+			AdjustStatNum(WRATH_STAT, ADV_EVENT_STAT_EASY)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/ardor_blossom.dm:34]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/ambling_pearl.dm [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+			return
+		if(3)
+			AdjustCurrency(ADV_EVENT_COIN_EASY)
+			AdjustStatNum(LUST_STAT, ADV_EVENT_STAT_EASY)
+		if(4)
+			AdjustHitPoint(-10)
+		if(5)
+			AdjustHitPoint(-4)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/ambling_pearl.dm:56]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/limbus_company/allaround_cleaner.dm [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+/datum/adventure_event/allaround_cleaner/EventChoiceFormat(obj/machinery/M, mob/living/carbon/human/H)
+	switch(cords)
+		if(1)
+			BUTTON_FORMAT(2, "YOU ARE SPECIAL", M)
+			BUTTON_FORMAT(3, "YOU ARE NOT SPECIAL", M)
+			return
+		if(3)
+			AdjustStatNum(GLOOM_STAT, ADV_EVENT_STAT_EASY)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/limbus_company/allaround_cleaner.dm:37]:
```suggestion
	return ..()
```

File: ModularTegustation/_adventure_console/adventure_events/general/jazz_end.dm [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+		if(2)
+			BUTTON_FORMAT(3, "LEAVE THE LAST TUNE", M)
+			BUTTON_FORMAT(4, "STOP THE RECORD", M)
+			return
+		if(3)
+			AdjustStatNum(SLOTH_STAT, ADV_EVENT_STAT_EASY)
+		if(4)
+			AdjustStatNum(GLOOM_STAT, ADV_EVENT_STAT_EASY)
+
+	. = ..()
```
Comment (Gboster-0) [inline ModularTegustation/_adventure_console/adventure_events/general/jazz_end.dm]:
```suggestion
	return ..()
```

**Review by Gboster-0 — COMMENTED** [review]
muh `return ..()`. Not exactly this PR's issue but should be adressed someday

**Comment by InsightfulParasite** [comment]
> muh `return ..()`. Not exactly this PR's issue but should be adressed someday

Gotcha Gotcha

**Pull request merged**